src/main/java/com/gregtechceu/gtceu/integration/kjs/recipe/GTRecipeSchema.java

```diff
@@ -679,6 +682,11 @@ public GTRecipeJS outputFluids(FluidStackJS... outputs) {
             return output(FluidRecipeCapability.CAP, (Object[]) outputs);
         }
 
+        public GTRecipeJS outputFluidsRanged(FluidStackJS output, int min, int max) {
+            return output(FluidRecipeCapability.CAP,
+                    IntProviderFluidIngredient.of(FluidIngredient.of(max, output.getFluid()), UniformInt.of(min, max)));
+        }
+
```

Comment:
also add a method that accepts an `IntProvider`:
```suggestion
        public GTRecipeJS outputFluidsRanged(FluidStackJS output, int min, int max) {
            return outputFluidsRanged(output, UniformInt.of(min, max)));
        }

        public GTRecipeJS outputFluidsRanged(FluidStackJS output, IntProvider range) {
            FluidStack stack = new FluidStack(output.getFluid(), (int) output.getAmount(), output.getNbt());
            return output(FluidRecipeCapability.CAP,
                IntProviderFluidIngredient.of(FluidIngredient.of(stack), range));
        }

```